app/src/lib/stores/app-store.ts

```diff
@@ -5488,3 +5507,11 @@ function isPullRequestAssociatedWithBranch(
     repositoryMatchesRemote(pr.head.gitHubRepository, remote)
   )
 }
+
+function getUntrackedFiles(
```

Comment:
Nit: could this method potentially belong on the WorkingDirectoryStatus class?

Reply:
hmm that's a good point. we have some similar helpers in `lib/status.ts`, so i've moved it there in 1e603eb